lib/helper.c

```diff
@@ -866,7 +867,8 @@ ipmi_start_daemon(struct ipmi_intf *intf)
 		exit(0);
 #endif
 
-	chdir("/");
+	ret = chdir("/");
+	assert(ret == 0);
```

Comment:
https://github.com/ipmitool/ipmitool/wiki/Coding-Standards
> Assertions
>
> `assert()` - don't; really, why would you?

Indeed, failing on an assertion is highly non-informative. Please properly check for the error and print the diagnostic message based on `errno`.

Reply:
sure.